WORKSPACE

```diff
@@ -466,3 +466,12 @@ maven_jar(
   artifact = "com.yahoo.pulsar:pulsar-client:jar:shaded:1.18"
 )
 # end Pulsar Client
+
+# for pex rules
+git_repository(
+    name = "io_bazel_rules_pex",
+    remote = "https://github.com/streamlio/bazel_rules_pex.git",
```

Comment:
@kramasamy Understood. Sounds good - but we need to set up mirror first. I will give a shipit now but we should make sure mirror is set up first (cc @huijunw since I'm on PTO.)